### PR TITLE
Task/bws 97 table placeholder

### PR DESCRIPTION
**Pull request by junojei**
localhost_3000_sc
localhost_3000_sc (1)
localhost_3000_sc (2)

### CODE REVIEW

**Review by Yaroslaww-1 — DISMISSED** [review]

File: frontend/src/components/common/table/table.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ const Table: FC<Props> = ({ columns, data, title, children, className }) => {
       data: data as Record<string, string>[],
     });
 
+  const hasData = data.length !== 0;
+
```
Comment (what1s1ove) [inline frontend/src/components/common/table/table.tsx]:
```suggestion
  const hasStrPlaceholder = Boolean(placeholder) // add a new prop. Should be an optional string
  const hasData = data.length !== 0;
  const hasPlaceholder = hasStrPlaceholder && !hasData; 

```

File: frontend/src/components/common/table/table.tsx [hunk 2]
```diff
@@ -60,6 +62,11 @@ const Table: FC<Props> = ({ columns, data, title, children, className }) => {
           })}
         </tbody>
       </table>
+      {!hasData && (
+        <div
+          className={styles.placeholder}
+        >{`No ${title?.toLowerCase()} to display`}</div>
+      )}
```
Comment (what1s1ove) [inline frontend/src/components/common/table/table.tsx]:
```suggestion
      {hasPlaceholder && (
        <div
          className={styles.placeholder}
        >{placeholder}</div>
      )}
```

**Review by what1s1ove — COMMENTED** [review]
centrify text
image

File: frontend/src/components/common/table/table.tsx (hunk 2 above)
Comment (what1s1ove) [reply frontend/src/components/common/table/table.tsx]: `placeholder` should be from props

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**